apps/ocsp.c

```diff
@@ -53,6 +53,18 @@ NON_EMPTY_TRANSLATION_UNIT
 #  define LOG_ERR       2
 # endif
 
+#if defined(OPENSSL_SYS_VXWORKS)
+/* setpgid() & fork() are not supported */
+int setpgid(pid_t pid, pid_t pgid)
+{
+    return 0;
+}
+
+pid_t fork(void)
+{
+    return (pid_t) -1;
+}
+#endif
```

Comment:
Please add a space between `#` and the directive keyword, so they aligned with the indentation above and blow.